libs/designsystem/data-table/src/sortable/sortable.component.spec.ts

```diff
@@ -0,0 +1,118 @@
+import { MockComponent } from 'ng-mocks';
+import { createHostFactory, Spectator } from '@ngneat/spectator';
+import { IconComponent } from '@kirbydesign/designsystem/icon';
+import { DesignTokenHelper } from '@kirbydesign/designsystem/helpers';
+import { TableSortableComponent } from './sortable.component';
+
+const getFontWeight = DesignTokenHelper.fontWeight;
+
+describe('TableSortableComponent', () => {
+  let spectator: Spectator<TableSortableComponent>;
+
+  const createHost = createHostFactory({
+    component: TableSortableComponent,
+    declarations: [MockComponent(IconComponent)],
+  });
+
+  beforeEach(() => {
+    spectator = createHost(`
+    <table class="kirby-table">
+      <th [sortable]="true">Data1</th>
+    </table>
+    `);
+  });
+
+  describe('by default', () => {
+    it('should create', () => {
+      expect(spectator.component).toBeTruthy();
+    });
+
+    it('should hide the sorting icon', () => {
+      const icon = spectator.query('kirby-icon');
+
+      expect(icon).toHaveComputedStyle({
+        visibility: 'hidden',
+      });
+    });
+  });
+
+  describe('sortable', () => {
+    it('should render a button, when sortable is true', () => {
+      const button = spectator.query('button');
+
+      expect(button).toBeTruthy();
+    });
+
+    it('should render a text, when sortable is false', () => {
+      spectator.setInput('sortable', false);
+
+      expect(spectator.element.firstChild.nodeName).toBe('#text');
```

Comment:
Consider checking nodetype instead and align with existing tests:
```suggestion
      expect(spectator.element.firstChild.nodeType).toBe(Node.TEXT_NODE);
```